packages/common/package.json

```diff
@@ -17,11 +17,8 @@
     "inversify": "^6.0.1",
     "jwt-decode": "^3.1.2",
     "lodash.merge": "^4.6.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
     "react-i18next": "^12.3.1",
-    "react-query": "^3.39.0",
-    "react-router-dom": "6.14.2",
+    "@tanstack/query-core": "^4.36.1",
```

Comment:
I think we also need to use https://www.npmjs.com/package/@tanstack/react-query in the other packages (that do use react-query). I don't think react-query v3 is compatible with query-core v4.

Reply:
I found that too much of a refactor for this moment, because it requires one (or 2) major updates. Can we postpone that? I've tested and found that react-query v3 with query-core v4 works well.

Reply:
Perhaps next maintenance sprint!